src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/entities/PrConfigCommandArguments.java

```diff
@@ -0,0 +1,58 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities;
+
+import hudson.util.DescribableList;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptions;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptionsDescriptor;
+

```

Comment:
Please add documentation for all new classes.
Make sure to explain their role.

Reply:
Done